src/org/vitrivr/cineast/core/data/MediaType.java

```diff
@@ -16,6 +16,7 @@
     IMAGE(1, "i", "image"),
     AUDIO(2, "a", "audio"),
     MODEL3D(3, "m", "3dmodel"),
+    ALL(98, "l", "all"),    //Used for legacy compatibility in job-scripts

```

Comment:
I'm not sure about this one, it might cause issues when an objectdescriptor of this type gets persisted for some reason. Wouldn't it be sufficient to make the parameter in the job file optional? Having it not set would be equivalent to setting it to 'all'.

Reply:
Implemented in 5c45630b685797e8deed1d3364cdaad80ac61517